src/components/Match/Laning/index.jsx

```diff
@@ -33,12 +33,18 @@ class Laning extends React.Component {
       match, strings, sponsorURL, sponsorIcon,
     } = this.props;
     const { laningColumns } = mcs(strings);
+    const annotatedPlayers = match.players.map((p) => {
+      const cs_t = p.lh_t.map((v, i) => v + p.dn_t[i])
+
+      return {...p, cs_t}
+    })
+
```

Comment:
Alright, another question.  (I have a lot of questions - thanks for bearing with me).

I would prefer to graph CS over time, rather than just LH.  It's possible to do that if I add this code.


In an ideal world, there would be a `Table` thing that we could pass something like, "the pairwise sum of lh_t and dn_t".  Or *EVEN* better, we could add something to the process in core (which I know where to add it), that would just put a cs_t field onto the `match.players` items.  Then, I guess we would need to re-process all of the matches that we want to see that sparkline thing on, and then ideally, we would do something where we show the `cs_t` if it is available, but if not, fall back to `lh_t`.

Reply:
If you could give a general 👍 or 👎 to this question, then I'll finish cleanup in general, and have this ready to merge this evening (or tomorrow morning).

Reply:
Since cs_t is a pure function of lh_t and dn_t, I think it's more space efficient to just compute it client side instead of adding it to the database in core.